docs/package_repos.rst

```diff
@@ -0,0 +1,68 @@
+Package repositories
+====================
+
+.. warning::
+    This document describes some work that is still in progress and may not be 100% applicable yet.
+
+SecureDrop publishes .deb and .rpm packages via apt and yum repositories, respectively.
+
+Each package repository is maintained in a specific Git LFS repository that is published to `Cloudflare's R2
+static hosting product <https://developers.cloudflare.com/r2/>`__. The Git repository contains the .deb and .rpm files, as well as the repository metadata.
+When a new commit is pushed, GitHub Actions (GHA) publishes the contents of the "public" folder to Cloudflare.
+We use `rclone <https://rclone.org/>`__ for this process to make it mostly vendor-neutral in case we
+need to switch providers in the future.
+
+There are three levels of package repositories, which correspond to different stages
+of the development process.
+
+For historical reasons, the apt repositories are on ``*.freedom.press`` while the yum respositories
+are on ``*.securedrop.org``.
```

Comment:
I think the note about domains should probably be higher in this document, in part for global context, but also so that the "There are three levels of package repositories..." line can then lead directly to the three subheadings where those are explained.